### PR TITLE
fix: improved structure and update to latest native release, fixes

### PR DESCRIPTION
**Pull request by rubin22**

### CODE REVIEW

File: build/ios/Pods.build/Release-iphonesimulator/Pods-Runner.build/dgph
Comment (ankit-uxcam) [inline build/ios/Pods.build/Release-iphonesimulator/Pods-Runner.build/dgph:1]: @rubin22 this file should be in gitignore

File: build/ios/Pods.build/Release-iphonesimulator/UXCam.build/dgph
Comment (ankit-uxcam) [inline build/ios/Pods.build/Release-iphonesimulator/UXCam.build/dgph:1]: @rubin22 this file should be in gitignore

File: build/ios/Pods.build/Release-iphonesimulator/flutter_uxcam.build/dgph
Comment (ankit-uxcam) [inline build/ios/Pods.build/Release-iphonesimulator/flutter_uxcam.build/dgph:1]: @rubin22 this file should be in gitignore

File: LICENSE [hunk 1]
```diff
@@ -1,10 +1,25 @@
-Copyright (c) 2019-2020, UXCam
-All rights reserved.
+Copyright (c) 2019-2020, UXCam. All rights reserved.
```
Comment (ankit-uxcam) [inline LICENSE:1]: @rubin22 please make it 2019-2023

Comment (rubin22) [reply LICENSE]: Done

File: build/ios/Pods.build/Release-iphonesimulator/flutter_uxcam.build/dgph
Comment (rubin22) [reply build/ios/Pods.build/Release-iphonesimulator/flutter_uxcam.build/dgph:1]: Added the build path to gitignore.

File: build/ios/Pods.build/Release-iphonesimulator/UXCam.build/dgph
Comment (rubin22) [reply build/ios/Pods.build/Release-iphonesimulator/UXCam.build/dgph:1]: Added the build path to gitignore.

File: build/ios/Pods.build/Release-iphonesimulator/Pods-Runner.build/dgph
Comment (rubin22) [reply build/ios/Pods.build/Release-iphonesimulator/Pods-Runner.build/dgph:1]: Added the build path to gitignore.

**Review by ankit-uxcam — APPROVED** [review]

**Pull request merged**